fe/fe-core/src/main/java/org/apache/doris/external/elasticsearch/EsNodeInfo.java

```diff
@@ -66,7 +71,7 @@ public EsNodeInfo(String id, Map<String, Object> map) throws DorisEsException {
             String address = (String) httpMap.get("publish_address");
             if (address != null) {
                 String[] scratch = address.split(":");
-                this.publishAddress = new TNetworkAddress(scratch[0], Integer.valueOf(scratch[1]));
+                this.publishAddress = new TNetworkAddress((useSslClient ? "https://" : "") + scratch[0], Integer.parseInt(scratch[1]));
```

Comment:
Actually I don’t want to replace here, can you consider a unified replacement in the `subphrase`?